### PR TITLE
Updated 'Structure of a project' section

### PR DESCRIPTION
**Pull request by staceydxw**
I've made some changes to the section of the Playbook that describes the different phases of a project. I've re-used content that we've included in sales proposals and tried to keep it brief.

### CODE REVIEW

File: playbook.md [hunk 1]
```diff
@@ -115,25 +115,25 @@ This isn't to say that we require clients to participate in everything. Clients
...
 
-At a high level, the purpose of each phase is:
+At a high level, we believe the purpose of each phase is:
 
-**Inception**: Talk to staff, leaders and stakeholders. Agree what the problem is, establish an initial strategy and working principles to help solve it
+**Inception**: Establish a single, multi-displinary team with the client. Bring everyone together for a kick-off session to agree principles for how we'll work together. Run a roadmapping workshop to define a set of goals and vision for the project. The roadmap also clearly defines dependencies and risks, and a list of things to learn or prove to form the basis of a user research plan
 
-**Discovery**: Talk to users, do research. Identify and document user needs, decide how to measure success
+**Discovery**: Carry out research to define users and understand their needs. Document findings, user needs and user journeys. Sometimes this phase also involves conducting technical discovery work to look at different technologies and solutions that might meet user's needs
```
Comment (evilstreak) [inline playbook.md]: Should this end `meet users' needs`?

File: playbook.md [hunk 2]
```diff
@@ -115,25 +115,25 @@ This isn't to say that we require clients to participate in everything. Clients
...
 
-**Alpha**: Prototyping, technical spikes and experiments. Make initial technology and service design decisions
+**Alpha**: Translate user needs into prototypes. Test prototypes with users and iterate them based on feedback. We also use the alpha phase to carry out technical spikes and experiments, and to make initial technology and service design decisions
 
-**Beta**: Design, development, usability testing. Build and launch a minimum viable product and learn how to scale it
+**Beta**: Build a working version of the service, based on what we learnt in alpha. Carry out regular usability testing of the service with users, and launch a minimum viable product. Make plans for supporting the service and continue iterating it.
 
-**Live**: Operate a new service, learn what works, continue to design and develop. Iteratively improve a live service
+**Live**: Operate the service, carry out regular user research and continue to itereate it. If new user needs are identified, we develop the service and launch new features to meet them.
```
Comment (evilstreak) [inline playbook.md]: Typo: `itereate` -> `iterate`

File: playbook.md (hunk 1 above)
Comment (staceydxw) [inline playbook.md]: yes it should

**Comment by staceydxw** [comment]
Going to merge this as I've made the required changes following Dom's feedback.

**Pull request merged**